modules/rosa-architecture.adoc

```diff
@@ -1,21 +1,31 @@
 //
 // * rosa_architecture/rosa_architecture_sub/rosa-architecture-models.adoc
-[id="rosa-architecture_{context}"]
-= ROSA architecture on public and private networks
 
-You can install ROSA using either a public or private network. Configure a private cluster and private network connection during or after the cluster creation process.
-Red Hat manages the cluster with limited access through a public network. For more information, see xref:../../rosa_architecture/rosa_policy_service_definition/rosa-service-definition.adoc#rosa-service-definition[ROSA service definition].
+[id="rosa-classic-architecture_{context}"]
+= ROSA Classic architecture
+
+In ROSA Classic, both the control plane and the worker nodes are deployed on the customer's VPC subnets.
+
+[id="rosa-classic-architecture-networks_{context}"]
+== ROSA Classic architecture on public and private networks
+
+With ROSA Classic, you can create your clusters on public or private networks.
+
+Customers can customize access patterns for their API server endpoint and Red Hat SRE management in the following ways:
+* Public - API server endpoint and application routes are internet-facing.
+* Private - API server endpoint and application routes are private. Private ROSA Classic clusters use some public subnets, but no control plane or worker nodes are deployed in public subnets.
+* Private with AWS PrivateLink - API server endpoint and application routes are private. Public subnets or NAT gateways are not required in your VPC for egress. ROSA SRE management uses AWS PrivateLink.
+
+The following image depicts the architecture of both public and private networks.
 
 .ROSA Classic deployed on public and private networks
 image::156_OpenShift_ROSA_Arch_0621_private_public_classic.png[ROSA deployed on public and private networks]
 
-If you are using {hcp-title-first}, you can create your clusters on public and private networks as well. The following images depict the architecture of both public and private networks.
+ROSA Classic clusters include infrastructure nodes where OpenShift components such as the ingress controller, image registry, and monitoring are deployed. The infrastructure nodes and the OpenShift components deployed on them are managed by ROSA Service SREs.
 
-.ROSA with HCP deployed on a public network
-image::ROSA-HCP-and-ROSA-Classic-public.png[ROSA with HCP deployed on a public network]
+The following types of clusters are available with ROSA Classic:
 
-.ROSA with HCP deployed on a private network
-image::ROSA-HCP-and-ROSA-Classic-private.png[ROSA with HCP deployed on a private network]
+* Single zone cluster - The control plane and worker nodes are hosted on a single availability zone.
 
-Alternatively, you can install a ROSA Classic cluster using AWS PrivateLink, which is hosted on private subnets only.
+* Multi-zone cluster - The control plane is hosted on three availability zones with an option to run worker nodes on one or three availability zones.
```

Comment:
This is not accurate. For multi-zone cluster,if set it to multi zone at the cluster creation then both control plane and default worker pool will be multi zone. Unless customer create additional machinepool with `--subnet` or `--availability-zone` set.

Reply:
Again, @bchandra-ocp - Can you please confirm?

Reply:
Is this inaccurate? 
https://docs.openshift.com/rosa/rosa_install_access_delete_clusters/rosa-aws-privatelink-creating-cluster.html#osd-aws-privatelink-about_rosa-aws-privatelink-creating-cluster

Reply:
The statement I provided is correct. 

Here I'm describing the architecture not how cluster is created by default. ROSA clusters can have machine pool in single AZ.  How the cluster is created by default (i.e., 3 AZs for control plane and 3 AZs for machine pools) and how machine pools can be created (3 AZs or 1 AZ) should be explained in the respective docs pages.

Reply:
Thank you, I will leave it as is.